doc/source/diagnostics/innodb_show_status.rst

```diff
@@ -224,10 +224,15 @@ between the current LSN and the LSN of the last completed checkpoint.
      :vartype: Numeric
      :scope: Global
 
-This variable shows the maximum allowed checkppoint age above which the redo
+This variable shows the maximum allowed checkpoint age above which the redo
 log is close to full and a checkpoint must happen before any further redo log
 writes.
 
+.. note:: 
+
+        This variable was removed in |Percona Server| 8.0.13-4 due to a change
+        in MySQL. The variable is identical to log capacity.
```

Comment:
"to log capacity."?

Reply:
From the description of the issue:
No longer prints maximum checkpoint age in SHOW ENGINE INNODB STATUS extensions due to it being made identical to log capacity after upstream 28072385

Thanks.

Reply:
Checked with Yura - he had nothing to add and said that Zsolt's comments covered everything.
Yura is ok with the current sentence - "The variable is identical to log capacity."